transforms/asset_url.js

```diff
@@ -22,11 +22,17 @@ module.exports = function( file, options ) {
 			var url = pathMapper( assetSrcAbsPath, function( srcDir ) {
 				return options.packagePathsToIds[ srcDir ] ? '/' + options.packagePathsToIds[ srcDir ] : null; // return val of outputDirPath needs to be absolute path
 			} );
-		
+
 			// all assets urls should be different than their paths.. otherwise we have a problem
 			if( url === assetSrcAbsPath )
 				return _this.emit( 'error', new Error( 'The file "' + assetSrcAbsPath + '" referenced from ##asset_url( "' + assetSrcPath + '" ) in file "' + file + '" is not an asset.' ) );
 
+			// TODO: shall we fix path-mapper instead of doing this?
+			var filename = path.basename(assetSrcAbsPath);
```

Comment:
this renaming seems to me outside of the intended scope of pathmapper, so I think this is in fact the best place for it, but the replace approach would break if filename occurs at unexpected locations in url. We should just figure out the last portion of url (whatever comes after the last "/") and replace that.